ace-setup-mq/remove-resources.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+# Functions
+printUsage() {
+    echo -e "\033[1;33mThis script will delete the specified objects:\033[0m"
```

Comment:
**question**: what is the purpose of \033?

Reply:
`\033[1;33m` is the starting sign and  `\033[0m` is the ending sign, text between them will be printed with specified color.

Reply:
👍